src/handbook/customer/sales/subscription-agreement-1.4.njk

```diff
@@ -125,7 +125,7 @@ PMB 24889, San Francisco, California 94104-5401 (or, if a different corporate en
 “User(s)” is defined as the unique and single Individual, employee, Contractor, or other third party individual or machine authorized by Customer (in accordance with this Agreement) that requires the provision of a seat within the admin platform, who are able to access the Software purchased under a Subscription, regardless of whether the User actually accesses or the frequency with which they access the Software. A User must be over the age of thirteen (13) years old.
 </p>
 <p>
-“Website” means FlowFuse’s website located at <a href="https://www.flowfuse.com">flowfuse.com</a>, <a href="https://www.flowforge.cloud">flowforge.cloud</a>,<a href="https://www.flowfuse.cloud">flowfuse.cloud</a> and all subdomains, and all content, services, documentation provided on the Website.
+“Website” means FlowFuse’s website located at <a href="https://www.flowfuse.com">flowfuse.com</a>, <a href="https://www.flowforge.com">flowforge.com</a>,flowfuse.cloud and all subdomains, and all content, services, documentation provided on the Website.
```

Comment:
@sumitshinde-84 @zackwasli why was this change merged? This is a legal definition, and should not be updated this way?

Reply:
It looked like these were simple link fixes, was trying to help while you were at the conference. Sorry if I merged something I shouldn't have!

Reply:
Not a problem, just making sure it's corrected